ynr/apps/candidates/tests/test_help_pages.py

```diff
@@ -8,15 +8,19 @@ def test_about_page(self):
 
     def test_about_page_links_to_privacy_policy(self):
         response = self.app.get("/help/about")
-        self.assertContains(response, '<a href="/help/privacy">here</a>')
+        self.assertContains(
+            response, '<a href="https://democracyclub.org.uk/privacy/">here</a>'
+        )
 
     def test_about_page_links_to_photo_policy(self):
         response = self.app.get("/help/about")
-        self.assertContains(response, '<a href="/help/photo-policy">here</a>')
+        self.assertContains(
+            response, '<a href="https://democracyclub.org.uk/privacy/">here</a>'
```

Comment:
I know I'm jumping into this after-the-fact now, but I'm really confused about this change.

I get that photos are personal data and the privacy policy contains some info about that, but the photo policy (i.e: "must be a clear, recent, colour headshot of the candidate" etc etc) addresses a different topic as it is about what we will accept as a submission.